docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Enforce Device Trust (Preview)
+description: Learn how to enforce trusted devices with Teleport
+videoBanner: gBQyj_X1LVw
+---
+
+<Admonition type="warning" title="Preview Note: Supported Resources">
+The device trust preview only supports SSH, Database and Kubernetes resources.
+Support for other resources is planned for upcoming Teleport versions.
+</Admonition>
+
+Resources protected by the device mode "required" will enforce the use of a
+trusted device, in addition to establishing the user's identity and enforcing
+the necessary roles. Furthermore, users using a trusted device leave audit
+trails that include the device's information.
+
+Device Trust enforcement can be configured with the following three modes of operation, represented
+by the `device_trust_mode` authentication setting:
+
+- `off` - disables device trust. Device authentication is not performed and
+  device-aware audit logs are absent.
+- `optional` - enables device authentication and device-aware audit, but does
+  not require a trusted device to access resources.
+- `required` - enables device authentication and device-aware audit.
+  Additionally, it requires a trusted device for all SSH, Database and
+  Kubernetes connections.
+
+### Prerequisites
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+
+
+## Role-based trusted device enforcement
+
+Role-based configuration enforces trusted device access at the role level. It
+can be configured with the `spec.options.device_trust_mode` option and
+applies to the resources in its `allow` rules. It
+works similar to [`require_session_mfa`](../guides/per-session-mfa.mdx).
+
+<Admonition type="tip" title="v13.3.6+">
+  Teleport version 13.3.6 and above supports preset `require-trusted-device` role to enforce Device Trust.
+  You have an option to either update existing role as referenced below, or use the preset role.
+  Ensure to update rules defined in preset `require-trusted-device` role as required.
+</Admonition>
+
+To enforce authenticated device checks for a specific role, update the role with
+the following:
+
+```diff
+kind: role
+version: v6
+metadata:
+  name: example-role
+spec:
+  options:
+    # require authenticated device check for this role
++   device_trust_mode: "required" # add this line
+  allow:
+    logins: ['admin']
+    kubernetes_groups: ['edit']
+    node_labels:
+      '*': '*'
+    ...
+
+```
+
+```code
+$ tctl create -f device-enforcement.yaml
+```
+
+```code
+$ tctl create -f admin.yaml
+```
+
+
+## Cluster-wide trusted device enforcement
+
+Cluster-wide configuration enforces trusted device access at the cluster level.
+
+<Admonition type="note" title="Default configuration">
+Enterprise clusters run in `optional` mode by default. Changing the mode to
+`required` will enforce a trusted device for all SSH, Database and Kubernetes
+accesses.
+</Admonition>
+
+
+<Admonition type="warning" title="Web UI">
```

Comment:
Double-admonition.

Reply:
The web ui is important one but I think the admonition block above this "Enterprise clusters run in `optional` mode by default. Changing the mode to `required` will enforce a trusted device for all SSH, Database and Kubernetes
accesses." can be added as a normal sentence.